config/application.rb

```diff
@@ -47,5 +47,7 @@ class Application < Rails::Application
     end
 
     config.middleware.use 'OliveBranch::Middleware'
+
+    config.filter_parameters << :form
```

Comment:
Is there any way to make this more specific? We may have forms in the future that don't need to be filtered. It looks like in Rails 5 you can specify the parent as well, maybe someone has written a port of that?
https://github.com/rails/rails/pull/13897